packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -97,17 +99,26 @@ export const TokensList: React.FC<TokensListProps> = (props) => {
               {sentenceCase(tokenCategory)}
             </AnchoredHeading>
             <Box marginBottom="space160" data-cy="tokens-table-container">
-              {tokens[tokenCategory].map(({name, value, comment}) => (
-                <TokenCard
-                  key={`token${name}`}
-                  category={tokenCategory}
-                  name={name}
-                  useCamelCase={useJavascriptNames}
-                  value={value}
-                  comment={comment}
-                  backgroundColor={backgroundColor}
-                />
-              ))}
+              {tokens[tokenCategory]
+                .sort((a, b) => {
+                  if (tokenCategory === 'font-weights') {
+                    const aValue: string = typeof a.value === 'string' ? a.value : a.value.toString();
+                    const bValue: string = typeof b.value === 'string' ? b.value : b.value.toString();
+                    return collator.compare(aValue, bValue);
+                  }
+                  return collator.compare(a.name, b.name);
+                })
```

Comment:
question: just wondering if we can run this once in an effect and save it to state or if this needs to run every re-render

Reply:
Alternate thought: what if we did this sorting on the design-token formatters level? I was tempted to do it there first but then saw the prior art.

Reply:
You can try that for the gatsby formatter export. If it doesn't work or is difficult to implement, you can alternatively consider a constant file somewhere in the website package that imports the theme, sorts it, and exports it out for this page to use.